buildSrc/src/main/kotlin/kotlinx.fuzz.example-module.gradle.kts

```diff
@@ -33,24 +35,18 @@ task("copyDependencies", Copy::class) {
     from(configurations.runtimeClasspath).into("${project.layout.buildDirectory.get()}/dependencies")
 }
 
-tasks.register<PrintTargetNames>("printTargetNames") {
+tasks.create<PrintTargetNames>("printTargetNames") {
```

Comment:
Why did you replaced `register` with `create`? AFAIK, `resiter` should be preferred, because it is lazy. So task wouldn't be configured unless needed